### PR TITLE
v1.0.0

### PR DESCRIPTION
**Pull request by nabobalis**
https://www.youtube.com/watch?v=KnyMPS2-x_c

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by nabobalis** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,47 +7,49 @@ package:
 source:
   fn: sunpy-{{version}}.tar.gz
   url: https://pypi.io/packages/source/s/sunpy/sunpy-{{ version }}.tar.gz
-  sha256: c14b6be4f4d705a235c26b1daa6dfb8136185e19ed1dc2e151561f602b3befb1
+  sha256: 8d9536ac4c779b0cee7f1cde4575c2bf5655ef67d67a0d25e39aa13a85ad75f1
 
 build:
   number: 0
-  script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
+  script: python -m pip install . --no-deps -vv
```
Comment (Cadair) [inline recipe/meta.yaml]: I think we still need this for `astropy_helpers` reasons right?

Comment (nabobalis) [reply recipe/meta.yaml]: Maybe?

**Comment by nabobalis** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -17,37 +17,37 @@ requirements:
   build:
     - {{compiler('c')}}  # [not win]
   host:
-    - python
+    - python >=3
```
Comment (Cadair) [inline recipe/meta.yaml]:
```suggestion
    - python >=3.6
```
???

**Comment by nabobalis** [comment]
@conda-forge-admin, please rerender

**Comment by nabobalis** [comment]
@conda-forge-admin, please rerender

**Comment by nabobalis** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 3]
```diff
@@ -12,16 +12,17 @@ source:
 build:
   number: 0
   script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
-
+  skip: true  # [py27]
```
Comment (Cadair) [inline recipe/meta.yaml]: might be more accurate to say `[py<3.6]`??

Comment (nabobalis) [reply recipe/meta.yaml]: it seems to already default to that? the re-render has not added any 3.5 builds.

Comment (Cadair) [reply recipe/meta.yaml]: although it will have the same effect because 3.6 is the min conda-forge version

**Comment by nabobalis** [comment]
@conda-forge-admin, please rerender

**Comment by Cadair** [comment]
@conda-forge-admin, please rerender

**Pull request merged**